### PR TITLE
Prevent MetaMask auto reconnect after disconnect

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/wallet/Wallet.tsx [hunk 1]
```diff
@@ -330,7 +335,7 @@ const Wallet = () => {
           if (typeof chainId === 'string' && !supportedChainIds.includes(parseInt(chainId))) {
             // TODO - We should probably request network change to network that is currently selected in the app network selector
             // Request user to change network to ethereum mainnet
-            requestNetworkChange(1);
+            requestNetworkChange(1, false);
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/wallet/Wallet.tsx:338]: what is `false`?

Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:338]: If you hover over function name in VSCode, you will see function signature with all params and their names.

Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:338]: when I do it, I'm seeing only one param `chainId: number`

Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:338]: Untitled

Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:338]: ah I was missing something

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**